README.md

```diff
@@ -9,13 +9,61 @@ as isolated as possible.
 The main purpose of bst is running CI/build processes in a somewhat
 deterministic fashion.
 
-# Usage
+## Usage
 
 ```
-$ bst <exe> <args...>
+$ bst [options] <exe> <args...>
 ```
 
-# Building from source
+See `man 1 bst` for more detailed information about how to use this
+program, including examples.
+
+## Why bst?
+
+While bst is a multi-purpose tool, its main purpose is to serve
+as a building block for larger container systems. In CI systems
+running lots of commands in rapid succession, the cost of spinning
+up Docker containers can be unacceptable. For instance, on an 8-core
+laptop, over 10 runs, it takes 1.15 seconds to run `/bin/true` on
+an Alpine Linux Docker image, while bst takes 0.07 seconds to setup
+and run the same program in an isolated environment.
+
+```
+$ perf stat -n -r 10 -- docker run --rm -it alpine true
+
+ Performance counter stats for 'docker run --rm -it alpine true' (10 runs):
+
+            1,1503 +- 0,0156 seconds time elapsed  ( +-  1,36% )
+
+$ perf stat -n -r 10 -- bst -r alpine /bin/true
+
+ Performance counter stats for 'bst -r alpine /bin/true' (10 runs):
+
+           0,07352 +- 0,00470 seconds time elapsed  ( +-  6,40% )
+```
+
+In other words, bst is not and does not want to be a replacement
+for Docker, but is meant to be used by tooling wanting low-overhead
+isolated environments.
+
+Another strong suit of bst is that, by design, it can be used unprivileged.
+bst uses well-defined semantics for user namespaces to give unprivileged
+users the rights to enter different environments in a safe and controlled
+manner.
+
+## Quickstart
+
+### Installing
+
+There are two ways to install bst: download a prepackaged binary, or building from source:
+
+#### Installing a binary package
+
+Go to the [release page](./releases) and download the binary archive of the latest release.
+
+Extract the archive into `/`. bst is installed into /usr/local.
+
+#### Building from source
 
 bst uses [Meson][meson] for its build system.
```

Comment:
It mightn't be necessary, but it could be instructive that, even if you change the install prefix, the `install` step *requires* sudo.